debian/libostree-1-1.symbols

```diff
@@ -204,6 +204,9 @@ libostree-1.so.1 libostree-1-1 #MINVER#
  ostree_repo_finder_get_type@LIBOSTREE_2017.8_EXPERIMENTAL 2017.8
  ostree_repo_finder_mount_get_type@LIBOSTREE_2017.8_EXPERIMENTAL 2017.8
  ostree_repo_finder_mount_new@LIBOSTREE_2017.8_EXPERIMENTAL 2017.8
+ ostree_repo_finder_override_add_uri@LIBOSTREE_2017.13_EXPERIMENTAL 2017.12-0endless2
```

Comment:
I think you need the `LIBOSTREE_2017.13_EXPERIMENTAL` version at the top of this file, too.

Reply:
Fixed.